code/modules/mob/living/carbon/human/human.dm

```diff
@@ -610,7 +610,7 @@
 		var/datum/data/record/R = find_record("name", perpname, GLOB.data_core.security)
 		if(R && R.fields["criminal"])
 			switch(R.fields["criminal"])
```

Comment:
I think you should add monitor to this with a += 1

Otherwise this PR is ready.

Reply:
thats amazing, thank you! i did think about having "search" tags add threat count so officers could come and pick em up but didn't have the thought of setting threat counts to monitor, that could however make people have a threat count that is above the minimum threat count required to arrset them. Tl:dr it could tip beepsky to arrest them imo, haven't looked into that yet